project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.2.8
+sbt.version=1.3.10
```

Comment:
Separate PR?

Reply:
Oops! That's what I thought. 😅Yeah, I will do another one with this.